### PR TITLE
Do more things

### PR DESCRIPTION
**Pull request by boatrite**

### CODE REVIEW

File: ansible/roles/common/files/files/authorized_keys [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDXctzutKtkSAjRZ4z50m7HPNNKFc9964WxpX7Sybs8SjgoM88K+P1OwaBDmX1YUpBab/lqKAuuDdeznIap0bopGCIYVlFGBMxf0csXv+Jp2ItrQhNPkCDyserI9W0YNCSAwJpYpaVQqxmorZafoioMb/fK7bGZ+mJ0mMRLvuuLcd34F98PI1eV78Xdu8w4MF5eRLibiTODrRwM29XB+Z52+jPCWSIY5DE1b+BRBtwMxMW2mw4KNdsiLhFT5VDiaYnAnZKyGw5gU8AJ0RHBkuYCTaXxx3LO8Ys1ttKY2I2R8DRO4swekC44NKJveWVbtovdGkWVYzlCJ6x/mRLrV8mzWwPOnhy3U5sYZr9+I3GXBhpsZHpvdM/YH7qhC3wrNcFupy6bjoeO5tE2rCEfM+PariZmbUgP0bJ89kq/5qj8lNu1rtp+w/2gKnZqoxdCZyKWvyR9z82Ico9nkxF03xJUummHzlemM3WtfUyw9Ua5zgFYWwr0zavj8kMrPx3+zE9+CaG7eCtkRYDHcX5RFh/wzrwjo3/kfKBn8+5XMpINegF0U0xlph5wC0OIJB2do+Nzr47GHa1Ity2Tan0WGBacWXBOQ0YiayE/h/4DhNjv00EbBVA7MV8xb/I7xIuWsSJgEHr8xReccTsMxYJZECx1hQBtgMiBx/Lk400t+W+Ovw== alex@tenforwardconsulting.com
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDNMfjXqD+TnTE+1fvjGZ05iEQYq121/Zwts0V+/M3jUvMHcQWbunQVASeofQZ7TAOgwfyNALhCf6dhxkxL9yMb4/ruEqOmcgvZiwWQxBepGe1nO/ElVRrocXj8qY4emmIsyLirVCTY2nf2IpSZmciIMrVbJ5xJYs7fL5+6by/cQkgYvCCQ4BFVzjNz21TkHZ0kEQdryWSH28ZFeFKPQ16xVp5npJ/p1kFbUyfCOQglU3HXSgs5Vp/HqKDhooJ+H/dMvOAipdSCxnHSiOwwABloO7HnSMyilqRaClXuKVej7KL0bLnzVSwc9GGp6lpmMSpewUvu78gTREvw3f9/e5q5 mark@mark.local
```
Comment (samsonasu) [inline ansible/roles/common/files/files/authorized_keys]: We shouldn't check this in here, we should put this in a .subspace/authorized_keys or something

File: ansible/roles/passenger/tasks/main.yml [hunk 2]
```diff
@@ -48,7 +48,7 @@
       dest: /etc/apache2/ssl/intermediate.crt
     sudo: true
 
-  - name: Create Rails Apache config (SSL Enabled)
```
Comment (samsonasu) [inline ansible/roles/passenger/tasks/main.yml]: This isn't the apache config it is the rails config for apache (e.g. it configures passenger, railsEnv, etc).

File: ansible/vars/aws.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+$ANSIBLE_VAULT;1.1;AES256
```
Comment (samsonasu) [inline ansible/vars/aws.yml]: This can't be checked in either

File: ansible/roles/passenger/tasks/main.yml (hunk 2 above)
Comment (boatrite) [inline ansible/roles/passenger/tasks/main.yml]: Oh, I see what you mean. This passenger role can be completely rails agnostic though. I put in the option to set the template src path which means we can template over any whatever-config for apache (in the intoxiclock case, a node config for apache). I'll reword that.

Comment (samsonasu) [inline ansible/roles/passenger/tasks/main.yml]: we probably wouldn't deploy node under apache we'd probably use nginx.  And passenger is a rails (rack) specific app server so i dont know what you mean by "rails agnostic.."  You woudn't use passenger without rails

Comment (boatrite) [inline ansible/roles/passenger/tasks/main.yml]: Right now intoxiclock update server is running node with apache. What does this passenger role even do? Why isn't this just under the apache role?

File: template/provision/ansible.cfg.erb [hunk 4]
```diff
@@ -1,6 +1,5 @@
 [defaults]
-
 inventory = hosts
 forks     = 10
-roles_path = <%= File.join(gem_path, 'ansible', 'roles') %>
+roles_path = <%= File.join(gem_path, 'ansible', 'roles') %>:/etc/ansible/roles
```
Comment (boatrite) [inline template/provision/ansible.cfg.erb]: issue

File: template/provision/host_vars/template.erb [hunk 5]
```diff
@@ -1,4 +1,4 @@
 ---
-ansible_ssh_host: <%= project_name %>-<%= @env %>
+ansible_ssh_host: <%= @hostname %>.10fw.net
```
Comment (boatrite) [inline template/provision/host_vars/template.erb]: issue not 10fw

**Pull request merged**